Cargo.toml

```
@@ -51,7 +51,7 @@ deref_mut = []
 display = ["syn/extra-traits"]
 error = ["syn/extra-traits"]
 from = ["syn/extra-traits"]
-from_str = []
+from_str = ["convert_case"]
```

This should fix the current compilation issue in the feature tests.
```suggestion
from_str = ["convert_case", "syn/extra-traits"]
```

---

Thanks, I just furoced pushed a change that bumps `syn` to 1.0.81 (which is the version in the lock file) and changed the code to use .is_empty() instead which means it does not need the extra traits

---

Bah looks like I also need to ditch the `x == from_str(x.to_string())` test as I can't `#derive[Display]` if the display feature is not set.